examples/tutorials/03_ready_to_use_model_tf.ipynb

```diff
@@ -57,7 +57,8 @@
    "source": [
     "[MNIST](http://yann.lecun.com/exdb/mnist/) is a dataset of handwritten digits frequently used as a baseline for machine learning tasks.\n",
     "\n",
-    "Downloading MNIST database might take a few minutes to complete."
+    "Downloading MNIST database might take a few minutes to complete.   \n",
+    "In your experiments you can use [Imagenette](https://github.com/fastai/imagenette)  and [CIFAR](https://www.cs.toronto.edu/~kriz/cifar.html) datasets as well."
```

Comment:
Referring to `fastai` and `torronto.edu` you give literally no clue on how to use these datasets with batchflow.